assembly/assembly-main/src/assembly/assembly.xml

```diff
@@ -76,12 +58,20 @@
     </dependencySets>
     <fileSets>
         <fileSet>
-            <directory>${project.build.directory}/dependency/</directory>
+            <directory>${project.build.directory}/dependency/assembly-che-tomcat8</directory>
             <outputDirectory>tomcat/</outputDirectory>
             <excludes>
                 <exclude>bin/*.bat</exclude>
             </excludes>
        </fileSet>
+        <fileSet>
+            <directory>${project.build.directory}/dependency/assembly-dashboard-war</directory>
+            <outputDirectory>tomcat/webapps/dashboard</outputDirectory>
+        </fileSet>
+        <fileSet>
+            <directory>${project.build.directory}/dependency/assembly-workspace-loader-war</directory>
```

Comment:
where does this content come from? I don't see these dependency folders under assembly/ or assembly-main/

Reply:
it gets unpacked at https://github.com/eclipse/che/pull/16802/files#diff-4c06d1de2892ec9f078d9036969ba103R103

Reply:
generated by maven-dependency-plugin https://github.com/eclipse/che/pull/16802/files#diff-4c06d1de2892ec9f078d9036969ba103R103-R117